core/CTestScript.cmake.in

```diff
@@ -20,8 +20,10 @@ set(CTEST_SOURCE_DIRECTORY @CMAKE_SOURCE_DIR@)
 set(CTEST_BINARY_DIRECTORY @CMAKE_BINARY_DIR@)
 
 ctest_start("Continuous")
-ctest_test(EXCLUDE_LABEL broken
-           PARALLEL_LEVEL 5
-           @REPEAT_UNTIL@
-           )
+ctest_test(EXCLUDE_LABEL broken|flaky PARALLEL_LEVEL 5)
+ctest_test(
+  INCLUDE_LABEL flaky
+  PARALLEL_LEVEL 5
+  @REPEAT_UNTIL@
+)
```

Comment:
Splitting this into two sets breaks Jenkins' ability to see the tests. It will only take the later run into account, which leads to a major regression is test evaluation.